apps/nova/src/components/TutorSettings/AccountSettings.tsx

```diff
@@ -0,0 +1,185 @@
+import { Controller, Label } from "@litespace/luna/Form";
+import { useFormatMessage } from "@litespace/luna/hooks/intl";
+import { Typography } from "@litespace/luna/Typography";
+import React, { useCallback, useMemo } from "react";
+import { UseFormReturn } from "react-hook-form";
+import { ITutorSettingsForm } from "@/components/TutorSettings/types";
+import {
+  useRequired,
+  useValidateEmail,
+  useValidatePassword,
+  useValidatePhoneNumber,
+} from "@litespace/luna/hooks/validation";
+import { governorates } from "@/constants/user";
+import { FullSwitch } from "@litespace/luna/Switch";
+import { useToast } from "@litespace/luna/Toast";
+
+const AccountSettings: React.FC<{
```

Comment:
We don't use the term `account`, we use `profile`. There is nothing special about the term `profile` other than it is better to stay consistent in our naming not to cause confusion. 

```suggestion
const ProfileSettings: React.FC<{
```